pkg/transformer/openshift/openshift.go

```diff
@@ -235,5 +241,74 @@ func (o *OpenShift) Deploy(komposeObject kobject.KomposeObject, opt kobject.Conv
 }
 
 func (o *OpenShift) Undeploy(komposeObject kobject.KomposeObject, opt kobject.ConvertOptions) error {
-	return errors.New("Not Implemented")
+	//Convert komposeObject
+	objects := o.Transform(komposeObject, opt)
+
+	// initialize OpenShift Client
```

Comment:
Same code for initializing openshift client is used in `Deploy` function. It might be better to move this to separate function (for example `(o *OpenShift) getOpenShiftClient (*Client, error)` ) and make Deploy and Undeploy call it.

Same thing  for Kubernetes client.